build, ui: fix make watch

couchand opened the pull request:
In #26148 the make watch task was missed.  This updates that to run with
the new webpack config.

Release note: None

couchand committed:
build, ui: fix make watch

In #26148 the make watch task was missed.  This updates that to run with
the new webpack config.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/26371)
<!-- Reviewable:end -->

vilterp reviewed (approved):
Tested locally; fixes the problem 👍

benesch commented on Makefile:
```diff
@@ -1198,7 +1198,7 @@ ui-watch-secure: export TARGET ?= https://localhost:8080/
 ui-watch: export TARGET ?= http://localhost:8080
 ui-watch ui-watch-secure: PORT := 3000
 ui-watch ui-watch-secure: $(UI_DLLS) pkg/ui/yarn.opt.installed
```
s/UI_DLLS/UI_CCL_DLLS/

(what I would give for some type safety in Make)

couchand committed:
build: fix ui make watch dependency on shared dlls

The make watch target runs the CCL build, so it should depend on the
CCL versions of the DLL files.

Release note: None
